src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  locales: 'http://localhost:4000'
```

Comment:
imho seems to be wrong here, you should define the path not an url, if you want to use it as a configuration.
Anyhow doing that on this way is much more confusing people reading how to do it.

Reply:
Thanks for the feedback. Don't know, I guess that would be an opinionated change. In my case I just followed gilsdav example, which is the reference today to configure the project with ssr, but with latest angular changes, with live reload including ssr and bringing it to the demo in the repo. You can have a look here: https://github.com/gilsdav/angular-universal-localize-router/blob/master/src/environments/environment.prod.ts

Won't change unless required by author :)

Reply:
That's correct. If you don't want to externalize the call (using http call), you need to specify a loader specifically for SSR part.